tests/unit/test_space.py

```diff
@@ -358,6 +394,7 @@ def test_box_bounds_attributes() -> None:
 )
 def test_box_contains_point(point: tf.Tensor) -> None:
     assert point in Box(tf.constant([-1.0, 0.0, -2.0]), tf.constant([2.0, 1.0, -0.5]))
+    assert Box(tf.constant([-1.0, 0.0, -2.0]), tf.constant([2.0, 1.0, -0.5])).contains(point)
```

Comment:
Would be slightly neater and easier to read if 
```python3
box = Box(tf.constant([-1.0, 0.0, -2.0]), tf.constant([2.0, 1.0, -0.5]))
assert point in box
assert box.contains(point)
```
(and similarly in other places which duplicate `point in` with `.contains(point)`)